src/renderers/dom/client/wrappers/ReactDOMTextarea.js

```diff
@@ -68,7 +69,7 @@ var ReactDOMTextarea = {
 
     // Always set children to the same thing. In IE9, the selection range will
     // get reset if `textContent` is mutated.
-    var nativeProps = assign({}, props, {
+    var nativeProps = assign({}, DisabledInputUtils.getNativeProps(inst, props), {
```

Comment:
I don't need the `{}` here - DisabledInputUtils returns a new object. @gaearon should the name of `getNativeProps` be changed to make that clearer - or do you think it is not necessary?